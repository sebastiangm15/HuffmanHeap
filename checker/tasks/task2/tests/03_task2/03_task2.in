10
7 SATZ
18 SATB
6 SAT7
11 SATX
15 SAT9
6 SATS
7 SATS
9 SATQ
9 SAT6
20 SAT0
5
110
1100
11001
011101
0100100
