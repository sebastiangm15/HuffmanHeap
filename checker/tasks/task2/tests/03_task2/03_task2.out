
SAT6 
SAT6 
SATS 
SATX SATX 
